client/app/src/accounts/AccountController.js

```diff
@@ -878,10 +852,23 @@
     };
 
     function sendArk(selectedAccount){
-      var passphrases = accountService.getPassphrases(selectedAccount.address);
-      var networks=self.net
+      var data={
+        fromAddress: selectedAccount ? selectedAccount.address: '',
+        secondSignature: selectedAccount ? selectedAccount.secondSignature: '',
```

Comment:
need to prefill with passphrase and second passphrase if saved (you get them with `accountService.getPassphrases(selectedAccount.address);`)